web/containers/application/Volunteer/Volunteer.js

```diff
@@ -0,0 +1,195 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Redirect } from 'react-router-dom';
+import { withFirebase } from 'react-redux-firebase';
+import Axios from 'axios';
+
+import Application from '../Application';
+import { changeVolunteerApplicationPage, changeVolunteerApplicationLastValidIndex, addVolunteerApplication, ACTION_TYPES } from '../../../actions';
+import propTypesTemplates from '../../../prop-types-templates';
+import { getFromFirestore } from '../../../services/firestore';
+import { initialState as volunteerApplicationInitialState } from '../../../reducers/entities/application/volunteer';
+
+import PageOne from '../../../components/application/volunteer/pages/PageOne/PageOne';
+import PageTwo from '../../../components/application/volunteer/pages/PageTwo/PageTwo';
+
+class VolunteerApplicationContainer extends React.Component {
+  static signUp(firebase, userCredentials) {
+    const auth = firebase.auth();
+    const { email, password } = userCredentials;
+    auth.createUserWithEmailAndPassword(email, password).then(() => {
+      console.log('user successfully created!');
+    }).catch((error) => {
+      console.err('failed to create user: ', error);
+    });
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = { isSubmitted: false };
+  }
+
+  submitApplication = (userCredentials, recaptchaResponse, onSubmitFailCallback) => {
+    const {
+      firebase,
+      volunteerApplication,
+      cancelApplication,
+      featureFlags: {
+        data: {
+          auth: {
+            enabled: isAuthEnabled,
+          },
+        },
+      },
+    } = this.props;
+
+    if (isAuthEnabled) {
+      this.signUp(firebase, userCredentials);
+    }
+
+    // include recaptcha token in payload for verification in cloud function
+    volunteerApplication.recaptchaResponse = recaptchaResponse;
+
+    if (!volunteerApplication.isSubmitted) {
+      // POST application data to cloud function for submission
+      Axios.post(
+        firebase.nwUtils.getFunctionUrl('submitApplicationVolunteer'),
+        volunteerApplication,
+        { headers: { 'Content-Type': 'text/plain' } }
+      ).then((res) => {
+        if (res.status === 200) {
+          console.log('Submitted volunteerApplication!');
+          this.setState({ isSubmitted: true });
+          cancelApplication();
+        } else {
+          onSubmitFailCallback();
+        }
+      }).catch(() => {
+        onSubmitFailCallback();
+      });
+    }
+  }
+
+  render() {
+    const {
+      featureFlags: {
+        isLoaded: isFeatureFlagsLoaded,
+        data: featureFlagsData,
+      },
+    } = this.props;
+
+    if (!isFeatureFlagsLoaded) return null;
+
+    const { isSubmitted } = this.state;
+    if (isSubmitted) return (<Redirect to="/success" />);
```

Comment:
`/success` is defined in `App.js` as being the SucessPage component from the hacker application. 

https://github.com/nwplus/nwhacks2019/blob/8d09824a0e6255f8b914fe6eba2b69ca62cd69c5/web/components/app/App.js#L19
https://github.com/nwplus/nwhacks2019/blob/8d09824a0e6255f8b914fe6eba2b69ca62cd69c5/web/components/app/App.js#L70

So right now the volunteer success page you created isn't being used. One fix I can think of is changing the current `/success` route (and any uses of it) to something like `/successHacker` and define a new route for the volunteer success page.

I'm mainly concerned about this because the hacker success page says "expect to hear back from us by the end of December" which is not true for volunteers.

Reply:
Good point. Its been fixed!